ui/admin/app/components/form/worker/index.hbs

```diff
@@ -0,0 +1,87 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @error={{@model.errors.base}}
```

Comment:
I don't think this is needed anymore, the form isn't using `@error`. It's likely a leftover artifact from a refactor.

Reply:
@ZedLi fixed